tests/python_client/testcases/test_search.py

```diff
@@ -2617,7 +2618,108 @@ def test_search_with_consistency_session(self, nq, dim, auto_id, _async):
                             check_items={"nq": nq,
                                          "ids": insert_ids,
                                          "limit": nb_old + nb_new,
-                                         "_async": _async}) 
+                                         "_async": _async})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.parametrize("collection_name", ["_co11ection$", "co11_$uper"])
+    @pytest.mark.parametrize("index_name", ["_1ndeX$", "In_$0"])
```

Comment:
Here this parametrize has some problems, will leads multiple collection with same collection name, which is invalid.

Reply:
> Here this parametrize has some problems, will create multiple collections with same collection name, which is invalid, need modify.

Done~!😎